R/util.R

```diff
@@ -41,6 +41,10 @@ pkg_file_arg <- function(...) {
   pandoc_path_arg(pkg_file(...))
 }
 
+pkg_file_lua <- function(...) {
+  pkg_file("rmd", "lua", ...)
+}
```

Comment:
I tend to follow @atusy's suggestion to export this function to encourage a consistent folder structure for Lua fitlers in other packages. Even if we export it, users don't have to follow this convention. They can choose to use a different structure.

However, if we were to export this, we probably should put the `lua` folder under the `inst/rmarkdown` directory instead of `inst/rmd`. This is to follow the convention of R Markdown templates in packages: https://bookdown.org/yihui/rmarkdown/template-structure.html

As an example, in **pagedown**, we may move the `resources/lua/` folder to `rmarkdown/`: https://github.com/rstudio/pagedown/tree/master/inst/resources/lua Then we can use `rmarkdown::pkg_file_lua('foo.lua', package = 'pagedown')`.

Anyway, this is not super important to me, and doesn't need to be done in this PR.

Reply:
I can see why it is useful. I'll do that; issue opened in #1903